tests/logictest/logictest.py

```diff
@@ -300,7 +309,8 @@ def execute(self):
                     f"Run query without session every statements, suite file path:{file_path}"
                 )
                 for state in get_statements(file_path, suite_name):
-                    self.execute_statement(state)
+                    for i in range(0, self.args.test_runs):
```

Comment:
test_runs should move to function https://github.com/datafuselabs/databend/blob/e14a82a0f44369c9cb2ec347fdcdd22baa832b25/tests/logictest/logictest.py#L315-L332   

One by one execute is rarely used, usually use batch_execute.

Reply:
batch_execute is implement in handlers , but call execute_statement finally.